### PR TITLE
[enriched/bugzilla] Add new field `whiteboard`

### PR DESCRIPTION
**Pull request by zhquan**
This code adds a new field to the enriched index that gives us
information about `whiteboard`.

Signed-off-by: Quan Zhou <quan@bitergia.com>

### CODE REVIEW

**Review by sduenas — CHANGES_REQUESTED** [review]
The PR looks ok but you need to update the CVS file with this information. I also think we should also add the same field for `bugzillarest` so we can have similar indices.

File: grimoire_elk/enriched/bugzillarest.py [hunk 1]
```diff
@@ -156,6 +157,9 @@ def get_rich_item(self, item):
             if 'changes' in history:
                 eitem['changes'] += len(history['changes'])
 
+        if issue['whiteboard'] != "":
```
Comment (sduenas) [inline grimoire_elk/enriched/bugzillarest.py:160]: Is this field in the item always? Not sure if that's going to create a `KeyError`.

Comment (zhquan) [reply grimoire_elk/enriched/bugzillarest.py:160]: In Bugzillarest the field is always there but not in Bugzilla. So to have the same behavior:

Option 1:
- Bugzilla: Add the field `whiteboard` only when the field does exist on the raw index.
- Bugzillarest: Add the field `whiteboard` only when the field is not empty("") on the raw index

Option 2:
- Bugzilla: Add the field `whiteboard` always and when the field doesn't exist set as empty("").
- Bugzillarest: Add the field `whiteboard` always

I choose the option 1

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**